dev_tools/packaging/set-dev-version.sh

```diff
@@ -19,49 +19,32 @@
 # the CIRQ_DEV_VERSION variable to a string that is the dev version
```

Comment:
Update this description to say that the script prints a new dev version. Also rename the script to `pick-dev-version` or `generate-dev-version-id` or similar to make this clearer.

Reply:
done